src/app/page.tsx

```diff
@@ -1,101 +1,93 @@
-import Image from "next/image";
+// this is the homepage
+
+import Header from '../components/header';  
+import '../styles/globals.css';
```

Comment:
use absolute imports, in case if the import path is long - we will consider alias-imports